lamin_cli/__main__.py

```diff
@@ -306,6 +304,9 @@ def save(filepath: str, key: str, description: str, registry: str):
 
     You can save a `.py` or `.ipynb` file as an {class}`~lamindb.Artifact` by
     passing `--registry artifact`.
+
+    To save renamed files, ensure that the tracking ID is specified in
```

Comment:
I don't think this wording is very clear. What is a "tracking ID"? That's a word we don't use anywhere.

Best is to use an example I think as we have on the `ln.track()` docs.

<img width="787" alt="image" src="https://github.com/user-attachments/assets/6083dfec-1cb5-4fe9-bc93-219bc8ec467f" />

Check out: https://docs.lamin.ai/lamindb.track

So, I think what we should do is somehow reuse that snippet and expose it here. Or paste a cross-reference so that people have a chance to understand what "tracking ID" (or a better word) means.